src/rdkafka.h

```diff
@@ -216,6 +216,22 @@ typedef enum rd_kafka_timestamp_type_t {
 
 
 
+/*!
+ * Enumerates the different rebalance protocol types.
+ *
+ * @sa rd_kafka_rebalance_protocol()
+ */
+typedef enum rd_kafka_rebalance_protocol_t {
+        RD_KAFKA_REBALANCE_PROTOCOL_NONE = 0,       /**< Rebalance protocol is
```

Comment:
if we allowed assignors to support more than one protocol type, these need to be flags. since we're skipping that (for now, probably forevr) i changed to enum. removing the number assignments.